style.css

```diff
@@ -0,0 +1,83 @@
+.main-header {
+  width: 730px;
+  height: 92px;
+  background-color: #3c3a39;
+  position: absolute;
+  left: 0;
+  top: 0;
+}
+
+.header {
+  display: flex;
+  color: white;
+  justify-content: space-between;
+}
+
+.name {
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  position: relative;
+  top: 50px;
+  left: 10px;
+}
+
+.menu {
+  position: relative;
+  right: 320px;
+  top: 50px;
+}
```

Comment:
- [ ] Kindly update the width of this element to prevent a horizontal scroll and replicate the design elegantly. 😎️
Currently, I'm seeing this to the right of the content.
<img src="https://user-images.githubusercontent.com/17243325/204980823-526a7c0a-db08-4428-b55b-4b9a11dbfea3.png" width="300">